### PR TITLE
Regenerated code to add corerct object types. (bunq/sdk_java#47)

### PR DESCRIPTION
**Pull request by OGKevin**
[//]: # (Thanks for opening this pull request! Before you proceed please make sure that you have an issue that explains what this pull request will do.
         Make sure that all your commits link to this issue e.g. "My commit. \(bunq/sdk_java#<issue nr>\)".
         If this pull request is changing files that are located in "com.bunq.sdk.model.generated" then this pull request will be closed as these files must/can only be changed on bunq's side.)
         
## This PR closes/fixes the following issues:
[//]: # (If for some reason your pull request does not require a test case you can just mark this box as checked and explain why it does not require a test case.)
 - Closes bunq/sdk_java#47
    - [x] Tested

### CODE REVIEW

**Comment by OGKevin** [comment]
39tests | 1failures | 0ignored | 38.142sduration

**Comment by OGKevin** [comment]
@patrickdw1991 Please :eyes: :popcorn:

File: src/main/java/com/bunq/sdk/model/generated/endpoint/AttachmentMonetaryAccount.java [hunk 1]
```diff
@@ -31,7 +31,6 @@
   /**
    * Object type.
    */
-  private static final String OBJECT_TYPE = "AttachmentMonetaryAccount";
```
Comment (patrickdw1991) [inline src/main/java/com/bunq/sdk/model/generated/endpoint/AttachmentMonetaryAccount.java]: Is it intended that the comment is still there when there are no constants to which the comment belongs to?

Comment (OGKevin) [reply src/main/java/com/bunq/sdk/model/generated/endpoint/AttachmentMonetaryAccount.java]: Ah no, the constant generator is not checking to see if it actually has constants to put underneath the doc block :sweat_smile:. Nice catch!

Comment (OGKevin) [reply src/main/java/com/bunq/sdk/model/generated/endpoint/AttachmentMonetaryAccount.java]: Fixed in generator, pending review.

**Comment by OGKevin** [comment]
@patrickdw1991 yours again please 👀

**Comment by patrickdw1991** [comment]
@OGKevin Approved, GOGOGO

**Pull request merged**